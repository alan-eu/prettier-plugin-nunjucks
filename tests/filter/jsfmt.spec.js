run_spec(__dirname, ["nunjucks"]);
