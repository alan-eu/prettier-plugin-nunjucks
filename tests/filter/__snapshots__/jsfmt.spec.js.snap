// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`filter.njk 1`] = `
====================================options=====================================
parsers: ["nunjucks"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
{% filter title %}
may the force be with you
{% endfilter %}

{% filter replace("force", "forth") %}
may the force be with you
{% endfilter %}

=====================================output=====================================
{% filter title %}
  may the force be with you
{% endfilter %}

{% filter replace("force", "forth") %}
  may the force be with you
{% endfilter %}

================================================================================
`;
